docs/api/records.rst

```diff
@@ -3,39 +3,324 @@
+        Host: localhost:8888
+        User-Agent: HTTPie/0.9.2
+        {
+            "data": {
+                "foo": "bar"
+            }
+        }
+
+  **Example Response**
+
+  .. sourcecode:: http
+
+        HTTP/1.1 201 Created
+        Access-Control-Expose-Headers: Backoff, Retry-After, Alert
+        Content-Length: 199
+        Content-Type: application/json; charset=UTF-8
+        Date: Thu, 18 Jun 2015 17:02:23 GMT
+        Server: waitress
+        {
+            "data": {
+                "foo": "bar",
+                "id": "89881454-e4e9-4ef0-99a9-404d95900352",
+                "last_modified": 1434646943915
+            },
+            "permissions": {
+                "write": [
+                    "basicauth_206691a25679e4e1135f16aa77ebcf211c767393c4306cfffe6cc228ac0886b6"
+                ]
+            }
+        }
+
+Replacing a record
+===================
+
+.. http:post:: /buckets/(bucket_id)/collections/(collection_id)/records/(record_id)
+
+    **Requires authentication**
+
+    Create or update a record in the collection.
+
+    **Example Request**
+
+    .. sourcecode:: bash
+
+        $ echo '{"data": {"foo": "baz"}}' | http put :8888/v1/buckets/blog/collections/articles/records/89881454-e4e9-4ef0-99a9-404d95900352 --auth="bob:" --verbose
+
+    .. sourcecode:: http
+
+        PUT /v1/buckets/blog/collections/articles/records/89881454-e4e9-4ef0-99a9-404d95900352 HTTP/1.1
+        Accept: application/json
+        Accept-Encoding: gzip, deflate
+        Authorization: Basic Ym9iOg==
+        Connection: keep-alive
+        Content-Length: 25
+        Content-Type: application/json
+        Host: localhost:8888
+        User-Agent: HTTPie/0.9.2
+
+        {
+          "data": {
+              "foo": "baz"
+          }
+        }
+
+    **Example Response**
+
+    .. sourcecode:: http
+
+        HTTP/1.1 200 OK
+        Access-Control-Expose-Headers: Backoff, Retry-After, Alert
+        Content-Length: 199
+        Content-Type: application/json; charset=UTF-8
+        Date: Thu, 18 Jun 2015 17:16:22 GMT
+        Server: waitress
+
+        {
+          "data": {
+              "foo": "baz",
+              "id": "89881454-e4e9-4ef0-99a9-404d95900352",
+              "last_modified": 1434647782623
+          },
+          "permissions": {
+              "write": [
+                  "basicauth_206691a25679e4e1135f16aa77ebcf211c767393c4306cfffe6cc228ac0886b6"
+              ]
+          }
+        }
+
+Updating a record
+=================
+
+.. http:patch:: /buckets/(bucket_id)/collections/(collection_id)/records/(record_id)
+
+    **Requires authentication**
+
+    Update a record in the collection. Specify only the fields to be modified,
+    all the rest will remain intact.
+
+    **Example Request**
+
+    .. sourcecode:: bash
+
+        $ echo '{"data": {"baz": "bar"}}' | http patch :8888/v1/buckets/blog/collections/articles/records/89881454-e4e9-4ef0-99a9-404d95900352 --auth="bob:" --verbose
```

Comment:
It's just a shortcut and the point is not actually showcasing that the thing is hapenning on localhost but what comes *after* the domain. I would rather keep it like that to avoid adding more information that seems less useful than the rest to the user, if you agree.